_notices/rsn0037.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 37 # should match notice number
+notice_pin: false # set to true to pin to notice page
+title: "Deprecation of `CentOS 7` in v24.06"
```

Comment:
Deprecation (this announcement) is happening for 24.04. Removal is happening in 24.06.
```suggestion
title: "Deprecation of `CentOS 7` in v24.04"
```